api/app/events/tests.py

```diff
@@ -47,7 +47,7 @@ def seed_static_data(self):
         db.session.add(self.test_form)
         db.session.commit()
 
-        db.session.flush()
+        # db.session.flush()
```

Comment:
Did this fix some error? Can remove if so?

Reply:
Yeah, it was for the dreaded detached instance error :)